src/helpers.jl

```diff
@@ -77,3 +77,26 @@ function sort_into_targets!(target, ws::NTuple{NT,W}, statss) where {NT,W}
     return target, ws, combine_stats(statss)
 end
 # three argument version for MPIData to be found in RMPI.jl
+
+# this is run during Rimu initialisation and sets the default
+"""
+    smart_logger(args...)
+Enable terminal progress bar during interactive use (i.e. unless running on CI or HPC).
+Arguments are passed on to `TerminalLoggers.TerminalLogger`. Undo with
+[`default_logger`](@ref).
+"""
+function smart_logger(args...; kwargs...)
+    if isa(stderr, Base.TTY) && (get(ENV, "CI", nothing) ≠ true)
+        Base.global_logger(TerminalLogger(args...; kwargs...)) # enable progress bar
+    end
```

Comment:
I'm not sure what's better here: this or `isinteractive`. Do we want scripts that don't redirect `stderr` to show progress bars?

Reply:
I did not know about `isinteractive`. However, the current setup also gives you a nice progress bar in the terminal when you start a script with
```> julia script.jl```
In this case `isinteractive` would return false.
I got the idea from https://github.com/timholy/ProgressMeter.jl#optional-use-of-the-progress-meter.

We can change it if we run into problems. With the testing I've done recently it seems to work fine.